### DISCUSSION TITLE
Quadro M4400 farming 0% load

### DISCUSSION THREAD

**Discussion by hajes** [Chia-Network/chia-blockchain#16237] [Support]
```
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 525.125.06   Driver Version: 525.125.06   CUDA Version: 12.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Quadro M4000        On   | 00000000:01:00.0 Off |                  N/A |
| 46%   34C    P8    10W / 120W |    478MiB /  8192MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A     87150      C   chia_harvester                    474MiB |
+-----------------------------------------------------------------------------+
```

All seems to be ok. No errors in chia log. I see signage points/lookups, but GPU shows 0% load. It looks like idling. 3510 C7 plots replotted with a new key for solo farming. Old 7000+ C0 plots have different key. No idea whether there is an issue with two different keys on the same harvester.

`chia farm summary` shows all plots, and seemingly all is ok.

<img width="1226" alt="Screenshot 2023-09-03 at 19 53 22" src="https://github.com/Chia-Network/chia-blockchain/assets/18664079/a3885729-4608-4acd-b65a-cb156ecc8646">

If I install tourist stuff `RTX 3060 Ti`, I see every 10s lookup spike...GPU core jumps up, wattage 80W. 

How can I check whether GPU is really used, please?

Of course `config.yaml` as described in Chia guide

```
harvester:
  decompressor_thread_count: 0
  parallel_decompressor_count: 1
  disable_cpu_affinity: false
  enforce_gpu_index: false
  gpu_index: 0
  max_compression_level_allowed: 7
  use_gpu_harvesting: true
```

**Comment by BrandtH22** [comment]
Hey @hajes just to clarify it sounds like you are seeing the expected GPU usage when the RTX 3060Ti is in the system but are not seeing the same GPU usage when the Quadro M4400 is in the system, is that correct? 

One good way to check if the GPU is being used properly is to run a chia plots check and look at the GPU usage during that time:
https://docs.chia.net/cli/#check

**Reply by BrandtH22** [reply]
Also, there is no real limit for the number of plotNFTs that you have on the machine but having multiple plot NFTs pointed to the same pool might cause issues (you would need to check with the specific pool).

**Reply by hajes** [reply]
I had a separate test rig with compressed plots only, and the behaviour was the same. Solo farming.

**Comment by hajes** [comment]
did `chia plots check`. In the beginning there was a gpu load 100 %...then

```
chia.plotting.check_plots        : ERROR    <class 'RuntimeError'>: GRResult is not GRResult_OK, received GRResult_Failed error in getting challenge qualities for plot xxxx.plot
chia.plotting.check_plots        : ERROR    	Proofs 4 / 30, 0.1333 Filepath: xxxx.plot
```

**Comment by hajes** [comment]
swapped to `RTX 3060 Ti`as suggested by guys in forum. Old GPUs do not work
